internal/pkg/service/appsproxy/config/config.go

```diff
@@ -53,8 +55,7 @@ func (c *Config) Normalize() {
 }
 
 func (c *Config) Validate() error {
-	errs := errors.NewMultiError()
-	return errs.ErrorOrNil()
+	return validator.New().Validate(context.Background(), c)
```

Comment:
Added missing validation.

Reply:
:eyes:

Reply:
Yes, you are right in `Streams`, but broken in `Templates`.
The `configmap.ValidateAndNormalize` should be used, the `Validate` method is optional, and is called by the `ValidateAndNormalize`.

Fixed: https://github.com/keboola/keboola-as-code/pull/1701/commits/e5bbf8a3f68a7df94ab5e3be6820fd79cc6fc231